sherlock/internal/models/test_data_test.go

```diff
@@ -15,7 +14,6 @@ func (s *modelSuite) TestBulkData() {
 	// Rather than iterating through the methods in order, use rand.Perm to effectively
 	// shuffle the order we process each index.
 	// https://pkg.go.dev/math/rand#Rand.Perm
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Is this not needed anymore on the new version?

Reply:
Correct! Got a linter error actually that prompted me to make this change, see [here](https://github.com/broadinstitute/sherlock/actions/runs/7412115470/job/20168179700?pr=413)